language/move-prover/spec-lang/src/translate.rs

```diff
@@ -4382,6 +4507,43 @@ impl<'env, 'translator, 'module_translator> ExpTranslator<'env, 'translator, 'mo
                 // Construct result.
                 let id = self.new_node_id_with_type_loc(&ty, loc);
                 self.set_instantiation(id, instantiation);
+
+                if let Operation::Function(module_id, spec_fun_id) = cand.oper {
+                    if self.translating_spec_fun {
+                        // Record the usage of function in specs, used later
```

Comment:
It's probably a good idea to make clear in such comments what functions you are talking about. Perhaps 'regular function' or 'Move function' for the one, 'spec fun' for the other. Here I'm not sure whether you are tracking spec funs or move funcs (and in the later case, the condition would be wrong.)